packages/jest-resolve/src/index.ts

```diff
@@ -414,21 +428,35 @@ class Resolver {
 
 const createNoMappedModuleFoundError = (
   moduleName: string,
-  updatedName: string,
-  mappedModuleName: string,
+  mapModuleName: (moduleName: string) => string,
+  mappedModuleName: string | Array<string>,
   regex: RegExp,
   resolver?: Function | string | null,
 ) => {
+  let mappedAs: string;
```

Comment:
Could perhaps use `pretty-format` to print this, eliminating the complicated custom stringify logic and the if-else?
Would look slightly different than now, with the `Array [` style, but that's probably fine, users see that style all the time.

Reply:
I tried `pretty-format`. 

<details>
<summary>Here is the result:</summary>

```js
Could not locate module ./style.css mapped as:
Array [
  "no-such-module",
  "no-such-module-2",
].

Please check your configuration for these entries:
{
  "moduleNameMapper": {
    "/\.(css|less)$/": "Array [
        "no-such-module",
        "no-such-module-2",
      ]"
  },
  "resolver": undefined
}
```
</details>

Note the `Array` prefix as you mentioned before.

I actually simplified code. Let me know if I still should use pretty-format. I'll need to add it as a dependency to resolve module, not sure if it's wanted

Reply:
@jeysal What do you think?

Reply:
I'd leave it as is, looks good this way 👍

Reply:
Nice! I'm marking this as resolved